modules/stochastic_tools/include/surrogates/PODReducedBasisTrainer.h

```diff
@@ -135,11 +135,11 @@ class PODReducedBasisTrainer : public SurrogateTrainer
   /// correlation matrices on the fly.
   void receiveObjects(
       ReplicatedMesh & mesh,
```

Comment:
While you're in here, you might as well forward declare this. I have changes that change the includes of your includes, and my branch doesn't compile without the forward declaration